src/utils/s3.js

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import { stringify } from 'qs';
+
+export default class S3Upload {
```

Comment:
Could the logic in this class be replaced with https://github.com/TTLabs/EvaporateJS?  The answer might be no; I don't have any experience with S3 uploads.

Reply:
I could see us maybe using that, however I don't think it would be much less source code for us, since we'd have to do a bunch of nonstandard configuration to get that library to work with Girder. Since the existing code works, I'd rather keep what's here for this PR, and then investigate using that library as a future topic.